src/anomalib/models/video/fuvas/README.md

```diff
@@ -0,0 +1,48 @@
+# [ICASSP 2025] FUVAS: Few-shot Unsupervised Video Anomaly Segmentation via Low-Rank Factorization of Spatio-Temporal Features
+
+## 📝 Description
+
+This folder contains the FUVAS video anomaly detection model which can support both transformer based backbone and CNN based backbone
+
+## 💡 Examples
+
+The following example shows how to use the FUVAS model to train on the ucsdped dataset.
+
+<details>
+<summary>Training the Fuvas model on UCSDped video dataset</summary>
+
+```python
+# Import the necessary modules
+from anomalib.data import UCSDped
+from anomalib.models import Fuvas
+from anomalib.engine import Engine
+
+# Load the avenue dataset, model and engine.
+datamodule = UCSDped()
+model = Fuvas()
+engine = Engine()
+
+# Train the model
+engine.train(model, datamodule)
+```
+</details>
+
+## Example running output
+![DRGSmethod](fuvas_anomalib_results.png)
```

Comment:
Can you add this markdown table instead, and remove `fuvas_anomalib_results.png`?

| Test metric | DataLoader 0 |
|-------------|-------------|
| frame_AUROC | 0.9135797023773193 |
| mean_F1Score | 0.9350237846374512 |
| pixel_AUROC | 0.9756277996063232 |